src/store/academy-context.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactNode, createContext } from 'react'
+import { AcademyDataType, useAcademyData } from 'components/hooks/use-academy-data'
+
+type AcademyProviderProps = {
+    children?: ReactNode
+}
+
+export type AcademyContextType = {
+    academy_data: AcademyDataType
+}
+
+export const AcademyContext = createContext<AcademyContextType>(null)
```

Comment:
```suggestion
type AcademyContextType = {
    academy_data: AcademyDataType
}

const AcademyContext = createContext<AcademyContextType>(null)
```
@mohammad-rework We don’t need to expose them to the whole codebase, You can just create a `useAcademy` abstraction.